index.html

```diff
@@ -0,0 +1,94 @@
+			<script  type="text/javascript" scr="script.js"></script>
+			<link rel="preconnect" href="https://fonts.googleapis.com">
+			<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+			<link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&family=Poppins:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+			<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&family=Poppins:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+
+			<link href="svg.link" type="text/css">
+			<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+			<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.css" integrity="sha512-FA9cIbtlP61W0PRtX36P6CGRy0vZs0C2Uw26Q1cMmj3xwhftftymr0sj8/YeezDnRwL9wtWw8ZwtCiTDXlXGjQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+			<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/brands.css" integrity="sha512-g9hNU97Dxb6qJZumYbKXC10tDP8J2rQdRxQoqona1LU0k0L8Hki+mOMHLbljf2TxSjgGl8f/PzUyGN3/XvA14w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+			<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous"/>
+      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/solid.css" integrity="sha384-Tv5i09RULyHKMwX0E8wJUqSOaXlyu3SQxORObAI08iUwIalMmN5L6AvlPX2LMoSE" crossorigin="anonymous"/>
+      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/fontawesome.css" integrity="sha384-jLKHWM3JRmfMU0A5x5AkjWkw/EYfGUAGagvnfryNV3F9VqM98XiIH7VBGVoxVSc7" crossorigin="anonymous"/>
+			<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/regular.min.css" integrity="sha512-k2UAKyvfA7Xd/6FrOv5SG4Qr9h4p2oaeshXF99WO3zIpCsgTJ3YZELDK0gHdlJE5ls+Mbd5HL50b458z3meB/Q==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+		</head>
+		<body>
+			<main>
+				<div class="header-background-mobile"></div>
+				<nav id="NavBar-status-light">
+					<time id="Time-Style">
+						<time class="Time">9:45</time>
+					</time>
+						<i class="fa-solid fa-wifi fa-xs"></i>	
+					<div class="sig-wf-bat">
+						<div class="signals">
+							<i class="fa-duotone fa-signal-bars-weak"></i>
+							<i class="fa-duotone fa-signal-bars-fair"></i>
+							<i class="fa-duotone fa-signal-bars-good"></i>
+							<i class="fa-duotone fa-signal-bars-strong"></i>
+						</div>
+						<div class="Battery">
+							<span class="Capacity"></span>
+							<span class="Border"></span>
+							<span class="Cap"></span>
+						</div>
+						<div class="wifi" >
+							<i class="fa-duotone fa-wifi-weak"></i>
+						</div>
+					</div>
+					</div>
+					<span class="rectangle"></span>
+				</nav>
```

Comment:
- [ ] Good job so far 👏. However, have some suggestions:

- [ ] First I think you don't need this status bar. It's just to simulate how is the phone but you don't need to add. 🙂 So, I would kindly ask you to remove it.

Reply:
@ridwanediallo Good morning, How can i revert a project